src/analyze/analyze.c

```diff
@@ -2366,6 +2388,31 @@ static int parse_argv(int argc, char *argv[]) {
 
                         break;
 
+                case ARG_ALIAS:
+                        if (!streq(strstrip(optarg), "")) {
```

Comment:
Is it safe to modify `optarg`??

Reply:
I believe it is, `optarg` is just a pointer in `argv`